frontend/src/features/workspace/components/DuplicateFormModal/DupeFormWizardProvider.tsx

```diff
@@ -24,13 +27,23 @@ export const useDupeFormWizardContext = (
   const { data: previewFormData, isLoading: isPreviewFormLoading } =
     usePreviewForm(
       formId,
-      // Stop querying once submissionData is present.
-      /* enabled= */ !!formId,
+      // Stop querying if formId does not exist or if it's in view template mode
+      /* enabled= */ !!formId || !isTemplate,
     )
 
+  const { data: templateFormData, isLoading: isTemplateFormLoading } =
+    useFormTemplate(
+      formId,
+      // Stop querying if formId does not exist or if it's not in preview mode
+      /* enabled= */ !!formId || isTemplate,
+    )
+
+  const data = templateFormData || previewFormData
+  const isLoading = isTemplateFormLoading || isPreviewFormLoading
```

Comment:
Suggestion:

```javascript

const { data, isLoading } = isTemplate ? useFormTemplate(formId, !!formId) : usePreviewForm(formId, !!formId)

Reply:
`React Hook "useFormTemplate" is called conditionally. React Hooks must be called in the exact same order in every component render` :(

Reply:
oh ya LOL lame, sure then hahaha that's fine